fastestimator/estimator/estimator.py

```diff
@@ -140,23 +140,41 @@ def _warmup(self):
             for epoch in signature_epochs:
                 ds_iter = self.pipeline.dataset_schedule[mode].get_current_value(epoch)
                 batch = next(ds_iter)
-                global_batch_size = self.pipeline.get_global_batch_size(epoch)
+                if self.distribute_strategy and mode == "train":
```

Comment:
do we still need to limit to train mode since we're doing the prediction concatenation now?